app/controllers/gi_controller.rb

```diff
@@ -11,6 +11,8 @@ def client
   end
 
   def scrubbed_params
-    params.except(:action, :controller, :format)
+    params.except(:action, :controller, :format).transform_values do |v|
+      v.encode!('UTF-8', invalid: :replace, undef: :replace, replace: '')
```

Comment:
I can add a guard that only does `encode!` if v is a string (or responds to encode, more specifically).

Rails treats the parameters as UTF-8, but it's possible to pass in a request param that has un-encodable characters in it. All the instances I see of this in the logs have a Windows user-agent. This request spec generates the exact stack trace we were seeing in Sentry.

If this was user data that we were storing for a form or something, I'd absolutely agree with you. For autocomplete and institution search, I'm less convinced that it's a big deal. 
- Many of the cases I could see of this in the logs were characters like a bullet point at the front or end of the parameter.
- There is a possibility of a character like ñ getting dropped, but I think the institution names are all normalized to not have any special characters so preserving such a character will never return any search results.